### PR TITLE
Fixes #1502. Prevents high CPU usage.

### LINKED ISSUE #1502: Application.UseSystemConsole = true consumes an entire cpu core on Ubuntu
If I `dotnet run` this demo code in an ubuntu20 desktop vm, it takes 100% of one cpu core:
```c#
using System;
using System.Diagnostics;
using System.Threading.Tasks;
using Terminal.Gui;
try
{
    Application.UseSystemConsole = true;
    Application.Init();
    Application.Top.Add(new Label("Hello World"));
    Application.Run();
}
finally
{
    Application.Shutdown();
}
```

Am I doing it wrong, or is this a bug?

### LINKED ISSUE #1520: NetDriver isn't updating the screen well.
The `Dialog` border is incomplete.

netdriver-update-screen

### PR DESCRIPTION
**Pull request by BDisp**
Fixes #1520.

### CODE REVIEW

**Review by tig — CHANGES_REQUESTED** [review]

File: Terminal.Gui/ConsoleDrivers/NetDriver.cs [hunk 1]
```diff
@@ -175,6 +175,7 @@ void CheckWinChange ()
 		void WaitWinChange ()
 		{
 			while (true) {
+				Thread.Sleep (10);
```
[the remark this answers is not in the document]
Comment (BDisp) [reply Terminal.Gui/ConsoleDrivers/NetDriver.cs:179]: I agree. Done. Thanks.

**Comment by tznind** [comment]
Should this be user configurable? i.e. a static property on the NetDriver class? Something like `WaitWinChangePollTimer`? That at least empowers developers to adjust it based on system settings/hardware or desired refresh times or whatever?

**Comment by BDisp** [comment]
@tznind `NetDriver` is internal, like `WindowsDriver` and `CursesDriver`, with the exception of `FakeDriver` for unit test. So, it must doed in the `Application`, but only for `NetDriver`, like the `AlwaysSetPosition` property, which I don't like.
In my opinion this setting is enough, so no much low and not making much difference and no much high and give time enough to detect any windows resize. If anyone more expertise can come with a better and appropriate solution, can submit a PR before or after this PR.

**Comment by tznind** [comment]
Cool, that sounds like a solid reason not to make it user configurable and that you have thought about it.   So I am happy :)

**Comment by TETYYS** [comment]
virtual medal to whoever releases this fix

**Review by tig — APPROVED** [review]
LGTM. Great work.

**Pull request merged**